File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> res(2);
        for(int left = 0; left < numbers.size(); left++){
            for(int right = left + 1; right < numbers.size(); right++){
                if(numbers[left] + numbers[right] == target){
                    res[0] = left + 1;
                    res[1] = right + 1;
                    return res;
                }
            }
       }
    }
};

File: Data Structures & Algorithms/binary-tree-right-side-view/submission-0.py
# Definition for a binary tree node.
# class TreeNode:
#     def __init__(self, val=0, left=None, right=None):
#         self.val = val
#         self.left = left
#         self.right = right

class Solution:
    def rightSideView(self, root: Optional[TreeNode]) -> List[int]:
        if not root:
            return []

        q = deque([root])
        res = []

        while q:
            q_len = len(q)
            r = None
            for x in range(q_len):
                node = q.popleft()
                if node:
                    r = node
                    q.append(node.left)
                    q.append(node.right)
            if r:
                res.append(r.val)

        return res